src/main/java/uk/gov/hmcts/reform/pdf/service/aspect/AuthorisationAspect.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.pdf.service.aspect;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.RequestHeader;
+import uk.gov.hmcts.reform.pdf.service.service.AuthorisationService;
+
+import java.lang.annotation.Annotation;
+
+@Aspect
+@Component
+public class AuthorisationAspect {
+
+    private boolean stop;
+
+    @Autowired
+    private AuthorisationService authorisationService;
+
+    private void parseAndAuthorize(RequestHeader header, String serviceAuthHeader) {
+        if (header.value().equals("ServiceAuthorization")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "ServiceAuthorization" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1132)